From 0c4d12762569df0991ce8ea17ec52ff35f3be463 Mon Sep 17 00:00:00 2001
From: Steve Grubb <sgrubb@redhat.com>
Date: Mon, 11 Sep 2023 17:29:53 -0400
Subject: [PATCH] remove use deprecated distutils python module
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Remove use deprecated distutils python module and use istead
sysconfig.get_path()

Signed-off-by: Tomasz Kłoczko <kloczek@github.com>
---
 configure.ac | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index faef0922a..ac0e5378f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,3 +1,4 @@
+
 dnl
 define([AC_INIT_NOTICE],
 [### Generated automatically using autoconf version] AC_ACVERSION [
@@ -168,15 +169,10 @@ else
 	PYTHON3_LIBS=`python3-config --libs 2> /dev/null`
 	PYTHON3_INCLUDES=`python3-config --includes 2> /dev/null`
 	AC_SUBST([PYTHON3_PREFIX], ['${prefix}'])
-	AC_SUBST([PYTHON3_EXEC_PREFIX], ['${exec_prefix}'])
-	PYTHON3_DIR=`$PYTHON3 -c "import distutils.sysconfig; \
-	print(distutils.sysconfig.get_python_lib(0,0,prefix='$PYTHON3_PREFIX'))"`
-	PYTHON3_EXECDIR=`$PYTHON3 -c "import distutils.sysconfig; \
-	print(distutils.sysconfig.get_python_lib(1,0,prefix='$PYTHON3_EXEC_PREFIX'))"`
+	PYTHON3_EXECDIR=`$PYTHON3 -c "import sysconfig; print(sysconfig.get_path('platlib'))"`
 	AC_SUBST(PYTHON3_CFLAGS)
 	AC_SUBST(PYTHON3_LIBS)
 	AC_SUBST(PYTHON3_INCLUDES)
-	AC_SUBST(python3dir, $PYTHON3_DIR)
 	AC_SUBST(py3execdir, $PYTHON3_EXECDIR)
 	fi
 fi
